TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html

```diff
@@ -59,12 +59,12 @@
 							<ul>
 								<f:for each="{documentNode.nodes}" as="relatedNode">
 									<li>
-										<f:if condition="{relatedNode.contextDocumentNode}">
+										<f:if condition="{relatedNode.accessible}">
```

Comment:
If the `relatedNode` is accessible, the `contextDocumentNode` also must be accessible, or did I understand something wrong there?

If there is no `contextDocumentNode`, the `relatedNode` is an orphan and should be removed by `node:repair`.
I could handle that more gracefully. Will add that.